### PR TITLE
fix: send all process.stdout/process.stderr writes through ipc

### PR DESCRIPTION
**Pull request by aleclarson**
In some cases, user-created logs may be written directly to `process.stdout` and `process.stderr` so this PR intercepts those too.

Another PR incoming that will print the Node.js logs in the main process and send them to Console.app with os_log on macOS/iOS.

### CODE REVIEW

File: npm/packages/@socketsupply/socket-node/index.js [hunk 1]
```diff
@@ -85,10 +108,10 @@ class API {
 
     if (data.length > MAX_MESSAGE_KB) {
       const len = Math.ceil(data.length / 1024)
-      process.stderr.write(
+      this.#writeStderr(
```
Comment (jwerle) [inline npm/packages/@socketsupply/socket-node/index.js:111]: @chicoxyzzy thoughts on dropping this? and the `512` limit?

File: npm/packages/@socketsupply/socket-node/index.js [hunk 2]
```diff
@@ -144,13 +167,13 @@ class API {
 
     if (s.length > MAX_MESSAGE_KB) {
       const len = Math.ceil(s.length / 1024)
-      process.stderr.write('WARNING: Sending large message to webview: ' + len + 'kb\n')
-      process.stderr.write('RAW MESSAGE: ' + s.slice(0, 512) + '...\n')
+      this.#writeStderr(
```
Comment (jwerle) [inline npm/packages/@socketsupply/socket-node/index.js:170]: @chicoxyzzy same here

Comment (chicoxyzzy) [reply npm/packages/@socketsupply/socket-node/index.js:170]: Yes, we can drop both. I don't know why we have such limit.

Comment (chicoxyzzy) [reply npm/packages/@socketsupply/socket-node/index.js:170]: @heapwolf maybe you know why?

**Review by jwerle — APPROVED** [review]

File: npm/packages/@socketsupply/socket-node/index.js (hunk 2 above)
Comment (jwerle) [reply npm/packages/@socketsupply/socket-node/index.js:170]: It was left over from the legacy module. We can drop it. @aleclarson could you remove the limit and warning?

Comment (chicoxyzzy) [reply npm/packages/@socketsupply/socket-node/index.js:170]: Let’s leave it for now. I will check if it’s necessary with #666

**Review by chicoxyzzy — APPROVED** [review]

File: npm/packages/@socketsupply/socket-node/index.js (hunk 2 above)
Comment (jwerle) [reply npm/packages/@socketsupply/socket-node/index.js:170]: SGTM

**Pull request merged**